specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+        "summary": "Get device credentials",
+        "x-ms-examples": {
+          "Get device credentials": {
+            "$ref": "./examples/devicecredentials_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceCredentials"
+            }
+          }
+        }
+      }
+    },
+    "/devices/{device_id}/modules": {
+      "get": {
+        "operationId": "Devices_ListModules",
+        "summary": "List the modules present in a device",
+        "x-ms-examples": {
+          "List modules": {
+            "$ref": "./examples/devices_modules_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Collection"
+            }
+          }
+        }
+      }
+    },
+    "/devices/{device_id}/modules/{module_name}/commands/{command_name}": {
+      "get": {
+        "operationId": "Devices_GetModuleCommandHistory",
+        "summary": "Get module command history",
+        "x-ms-examples": {
+          "Get command history within module": {
+            "$ref": "./examples/modules_commandshistory_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/ModuleName"
+          },
+          {
+            "$ref": "#/parameters/CommandName"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceCommandCollection"
+            }
+          }
+        }
+      },
+      "post": {
+        "operationId": "Devices_ExecuteModuleCommand",
+        "summary": "Execute a module command",
+        "description": "Execute a command on a module.",
+        "x-ms-examples": {
+          "Send command to module": {
+            "$ref": "./examples/modules_commands_send.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/ModuleName"
+          },
+          {
+            "$ref": "#/parameters/CommandName"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device command body.",
+            "schema": {
+              "$ref": "#/definitions/DeviceCommand"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
Here's a case where I'd expect a 202 "accepted" response, given this is a POST verb and it is long-running.

Reply:
POST command is "creating" a command request entity (the intent being that, in the future, you'll be able to list multiple of these entities, a.k.a. the command history), thus 201 (Created).